server/schedule.py

```diff
@@ -85,6 +85,19 @@ def _next(self, cron, after, user):
         except ValueError as e:
             raise ContentError(e)
 
+    def _previous(self, cron, before, user):
+        """Finds the previous time matching the cron expression."""
+
+        try:
+            cron = self._sun.rewrite_cron(cron, before, user)
+        except DataError as e:
+            raise ContentError(e)
+
+        try:
+            return croniter(cron, before, user).get_prev(datetime)
```

Comment:
Passing `user` as the third argument to `croniter` looks accidental here.

Reply:
Ahh yeah, good catch. Not sure what I was thinking.